R/osmdata.R

```diff
@@ -217,10 +226,13 @@ get_osm_river <- function(bb, river_name, crs = NULL, force_download = FALSE) {
   river_centerline <- osmdata_as_sf("waterway", "river", bb,
                                     force_download = force_download)
   river_centerline <- river_centerline$osm_multilines |>
-    dplyr::filter(.data$name == river_name) |>
+    # filter using any of the "name" columns (matching different languages)
```

Comment:
This should allow for checking multiple columns with `name` in as header (i.e. multiple languages) - we could integrate this and the similar filtering used for the city boundary in a separate function, potentially including partial name matching. This should help with #67